recipes/tl-expected/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "1.0.1":
```

Comment:
Only use official releases: https://github.com/TartanLlama/expected/releases

The latest available is 1.0.0

Reply:
1.0.1 fixes a bug in tl-expected that I've run into in a codebase: https://github.com/TartanLlama/expected/issues/61

I'll see if the author will label 1.0.1 for official release.

Reply:
Hi, sorry for the delay on this.

The author is very busy at the moment, but I will take responsibility for a fork that contains a 1.0.1 release.  (I guarantee that that will be the only difference between my fork and upstream.)   This recipe can be updated with the official repository's release later.

Reply:
I will start with the 1.0.0 first and wait a little bit until the author releases an official 1.0.1. Otherwise would mess up with the version tagging and produce trouble on future users

Reply:
I started with 1.0.1 because 1.0.0 contains a bug that I've hit in production code.  I am okay with switching this back to tl-expected at 1.0.0, but I won't be able to use the conan-center-index package :stuck_out_tongue:  

However, I agree that that's not a requirement.

Reply:
ok, as an alternative for the "1.0.1" version, let's use a specific commit of the original repo and version it with the date (https://bincrafters.readthedocs.io/en/latest/using_packages.html#packages-without-official-releases) That way we are safe and it will not mislead users when the real 1.0.1 comes out. And add the 1.0.0 version as well please

Reply:
That sounds good to me.  I'll see if I can get that going this evening.

Reply:
72f6e1015c4951fad8e2386862585329155af958 modifies this PR to supply `tl-expected/1.0.0` and `tl-expected/20190710`, with 2019-07-10 being the date of the commit that fixes the aforementioned bug (and bumps tl-expected's internal version number to 1.0.1).